helper/staking/staking.go

```diff
@@ -134,12 +156,22 @@ func PredeployStakingSC(
 		return nil, fmt.Errorf("unable to generate DefaultStatkedBalance, %w", err)
 	}
 
+	//Validate min and max validators number
+	if err = validateMinMaxValidatorsNumber(params.MinValidatorCount, params.MaxValidatorCount); err != nil {
+		return nil, err
+	}
```

Comment:
This validation should not be done at this level in the predeploy pipeline.

Please consider moving this check outside this method 🙏

Reply:
fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a